caffe2/core/tensor_factories.cc

```diff
@@ -0,0 +1,12 @@
+#include "tensor_factories.h"
+
+namespace caffe2 {
+
+Tensor empty(const vector<int64_t>& dims, const at::TensorOptions& options) {
```

Comment:
so is the plan to merge this empty() with the actual at::empty? what prevents us from doing so?

Reply:
we should be able to do that after the two Tensor classes are merged :)